pkg/modifycontroller/modify_status.go

```diff
@@ -57,13 +56,14 @@ func (ctrl *modifyController) markControllerModifyVolumeStatus(
 			[]v1.PersistentVolumeClaimCondition{pvcCondition})
 	}
 
-	updatedPVC, err := util.PatchClaim(ctrl.kubeClient, pvc, newPVC, true)
+	updatedPVC, err := util.PatchClaim(ctrl.kubeClient, pvc, newPVC, false)
```

Comment:
can you add a comment to this line? why was this changed from `true` to `false`? Also - I always find a comment of the form of - `false /* addResourceVersionCheck */` useful, next to a boolean value to explain what effect this has on the function.